docs/tutorial/recent-documents.md

```diff
@@ -83,6 +83,21 @@ following code snippet to your menu template:
 }
 ```
 
+Make sure the application menu is added on application 'whenReady' and not before, or 'Open Recent' will be disabled:
+
+```javascript
+const { app, Menu } = require('electron');
```

Comment:
Our linter requires you remove these semicolons - it's stalled out it looks like right now but if you push another commit it'll fail. You can verify locally by running `npm run lint:docs` which will show:

```
electron on git:patch-1 ❯ npm run lint:docs                                  10:38AM

> electron@14.0.0-nightly.20210430 lint:docs
> remark docs -qf && npm run lint:js-in-markdown && npm run create-typescript-definitions && npm run lint:docs-relative-links && npm run lint:markdownlint


> electron@14.0.0-nightly.20210430 lint:js-in-markdown
> standard-markdown docs

⠋


   tutorial/recent-documents.md
         89:42:    Extra semicolon.
         93:2:     Extra semicolon.
         97:32:    Extra semicolon.
         98:3:     Extra semicolon.

```

Reply:
Thanks! I'll update the PR.

Reply:
@codebytere  The commit is green now 👍